index.js

```diff
@@ -156,26 +178,65 @@ const ScrollableTabView = React.createClass({
          }}
          ref={(scrollView) => { this.scrollView = scrollView; }}
          {...this.props.contentProps}>
-         {this._children().map((child, idx) => {
-           return <View
-             key={child.props.tabLabel + '_' + idx}
-             style={{width: this.state.containerWidth, }}>
-             {child}
-           </View>;
-         })}
+           {scenes}
         </ViewPagerAndroid>
       );
     }
   },
 
+  _composeScenes() {
+    let scenes = [];
+    this._children().forEach((child, idx) => {
+      let key = this._makeSceneKey(child, idx);
+      let selected = false;
+
+      // if key doesnt exist(typically before each tab is first mounted) then put a dummy child instead of real
+      // this ensures # of pages remain the same , leaving scrolling and click tab effect to be handled as intended
+      if(!this._keyExists(this.state.sceneKeys, key)) {
+        let scene = (
+          <SceneComponent
```

Comment:
I'm trying to follow khan academy style guide 
https://github.com/Khan/style-guides/blob/master/style/react.md#align-and-sort-html-properties 
if it is not difficult could you please to fix indents?